lib/hammer_cli_foreman/virtual_machine.rb

```diff
@@ -0,0 +1,78 @@
+
+module HammerCLIForeman
+
+  class VirtualMachine < HammerCLIForeman::Command
+
+    resource :compute_resources
+    command_name 'virtual_machine'
+    desc _("View and manage compute resource's virtual machines")
+
+
+    class AvailableVirtualMachinesCommand < HammerCLIForeman::ListCommand
+
+      resource :compute_resources, :available_virtual_machines
+      command_name 'list'
```

Comment:
If the command is `ListCommand` already, there is no need to explicitly specify command's name. Applies for command definitions below as well.